app/components/e-radio.vue

```diff
@@ -0,0 +1,250 @@
+   */
+  export default {
+    name: 'e-radio',
+    // components: {},
+    mixins: [formStates],
+    inheritAttrs: false,
+
+    model: {
+      /**
+       * Changes v-model behavior and use 'selected' instead of 'value' as prop.
+       * Avoids conflict with default value attribute.
+       */
+      prop: 'selected',
+      event: 'change',
+    },
+
+    props: {
+      /**
+       * Adds value attribute.
+       */
+      value: {
+        required: true,
+        type: String,
+      },
+
+      /**
+       * Adds displayed value of the radio element.
+       */
+      displayName: {
+        required: true,
+        type: String,
+      },
+
+      /**
+       * Adds name attribute.
+       */
+      name: {
+        required: true,
+        type: String,
+      },
+
+      /**
+       * Adds selected attribute to be used by v-model.
+       */
+      selected: {
+        default: '',
+        type: String,
+      }
+    },
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      internalValue: {
+        get() {
+          return this.selected;
+        },
+        set(value) {
+          /**
+           * Fired on select of radio button.
+           *
+           * @event change
+           * @type {String}
+           */
+          this.$emit('change', value);
+        }
+      },
+
+      modifiers() {
+        return {
+          ...this.stateModifiers,
+          selected: this.internalValue === this.value,
+        };
+      }
+    },
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      /**
+       * Emits input to parent component
+       *
+       * @param   {String}  event   Field input
+       */
+      onChange(event) {
+        /**
+         * Emits change event.
+         *
+         * @event change
+         * @type {String}
+         */
+        this.$parent.$emit('change', event.target.value);
+      },
+    },
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .e-radio {
+    position: relative;
+
+    &__field {
+      opacity: 0;
+      position: absolute;
+      left: -9999px;
+    }
+
+    &__fake-button::after {
+      position: relative;
+      content: '';
+    }
+
+    &__fake-button {
+      -webkit-appearance: none;
+      background-color: $color-grayscale--1000;
+      border: 1px solid $color-grayscale--400;
+      border-radius: 10px;
+      display: inline-block;
+      position: absolute;
+      width: 16px;
+      height: 16px;
+      top: 4px;
+      left: 0;
+      cursor: pointer;
+    }
+
+    &__label-name {
+      padding-left: $spacing--25;
+      cursor: pointer;
+      color: $color-grayscale--400;
+    }
+
+    &__label {
+      cursor: pointer;
```

Comment:
- [ ] I would overwrite here the default `label` margin-bottom to 0, because the vertical spacings shouldn't be handled inside this component (Can be used vertical and horizontal)

Reply:
@mob8607 FYI (I've already changed it), I thought that the spacings are never inside the component. So it looks probably not nice if you rendere a few of them in single rows but you have to add the styling where you loop it - so you can use them horizontal and vertical

Reply:
Ok, got it. I forgot to add it on the wrapper so it looks better in the demo.